src/renderer/Voice.tsx

```diff
@@ -1194,6 +1209,13 @@ const Voice: React.FC<VoiceProps> = function ({ t, error: initialError }: VoiceP
 		const tempTalking = { ...otherTalking };
 		let talkingUpdate = false;
 		for (const player of otherPlayers) {
+			// bug with TOH where all players are considered alive during the ejection screen. This is a workaround
+			if (checkIfPlayerWasDeadAtStartOfMeeting(player.ptr) && !player.isDead) {
+				player.isDead = true;
+			}
```

Comment:
O(n^2), this is what lead me to look for other ways to do it and found the otherDead variable previously mentioned. It uses an indexed type so instead of iterating the entire player list for every player you can directly look it up. I know the performance difference is probably negligible but it _should_ be slightly faster to use a look up type and I believe this is ran for basically every frame of the game.